client/src/components/CategoryBox.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import "../styles/navbar.css";
+import "../styles/category-box.css"
+
+function CategoryBox(props) {
+    return (
+        <div id="box-container">
+            <div id="content-container">
```

Comment:
Is there a reason why you set the background of the div as the image as opposed to using the `<img src="" alt="">` tag? This way you can add styling to it pretty easily. I imagine this would be difficult when there are multiple images in the future..

Reply:
I can change that in just a sec. I was just making the whole background an image because I assumed the text wouldn't go over it otherwise.

Reply:
I just changed it to be an img tag instead of the background being the image